doc/guidelines_matrix.md

```diff
@@ -1,21 +1,22 @@
 # Guidelines Matrix
 
-Es gibt folgende Matrix-Räume:
+Es gibt folgende offizielle Matrix-Räume:
 
 - FOSS-AG Hauptkanal (öffentlich)
   - Es gibt einen Admin
-  - Jedes aktive Mitglied kann Moderator-Rechte erhalten
+  - Jedes aktive Mitglied kann auf Anfrage Moderator-Rechte erhalten.
   - Themen:
-    - FOSS-AG Aktivitäten
+    - Allgemeine FOSS-AG Aktivitäten
     - FOSS-Themen
     - Vorstellung von Teams, Events, etc.
-- FOSS-AG Orga (nur aktive Mitglieder)
+- FOSS-AG Organisation (öffentlich, im Hauptkanal verlinkt)
```

Comment:
Wenn das so umgesetzt wird, dann in der Form dass ein neuer Kanal erstellt wird.

Reply:
lang oder kurzform?

Reply:
und was mir gerade noch so auffällt, da steht aktives Mitglied drin.

Reply:
Da dieser PR sich nur auf die Matrix-Räume bezieht, steht hier noch das von 'aktiven'.